tree/dataframe/inc/ROOT/RResultPtr.hxx

```diff
@@ -302,6 +307,18 @@ public:
       return *this;
    }
 
+    /// Return a pointer to the result, releasing its ownership and leaving this object empty.
+    T* &Release()
```

Comment:
```suggestion
    T* Release()
```

Reply:
this removes the compilation warnings

Reply:
@eguiraud  Previous Version of the commit wasn't pushed
I have fixed it now.